code/datums/status_effects/buffs/bioware/_bioware.dm

```diff
@@ -0,0 +1,28 @@
+/**
+ * ## Bioware status effect
+ *
+ * Simple holder status effects that grants the owner mob basic buffs
+ */
+/datum/status_effect/bioware
```

Comment:
Should this use ``on_remove_on_mob_delete``?

Reply:
It's not necessary. 

The mob's being deleted anyways, so it's not like we need to remove the traits from the mob, since... it's being deleted. 

(in fact, it would runtime because physiology would be null'd by the time these are removed)